vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationMain.java

```diff
@@ -138,20 +136,18 @@ private ExperimentId activeExperiment(ProjectId parsedProjectId) {
 
   private void setContext(Context context) {
     experimentContentComponent.setContext(context);
-    experimentSupportComponent.projectId(context.projectId().orElseThrow());
-    experimentSupportComponent.setExperiments(
-        experimentInformationService.findAllForProject(context.projectId().orElseThrow()));
+    experimentSupportComponent.setContext(context);
     experimentSupportComponent.setSelectedExperiment(context.experimentId().orElseThrow());
     projectNavigationBarComponent.projectId(context.projectId().orElseThrow());
     this.context = context;
   }
-
-
   private void addListeners() {
     experimentSupportComponent.addExperimentSelectionListener(
         event -> routeToExperiment(event.getSource().experimentId()));
     experimentSupportComponent.addExperimentCreationListener(
         event -> routeToExperiment(event.experimentId()));
+    experimentContentComponent.addExperimentEditListener(
```

Comment:
Why reroute? We are already at this experiment. This leads to an additional loading of the experiment.

Reply:
Yes and that is intended so the ExperimentListComponent also shows the changed experimentinformation.

Reply:
Then I would argue that the parent (main component) should listen to context changes and propagate those to the support section.

Reply:
Alright I added a dedicated listener within the experimentdetailscomponent and propagate the new context to the support component once the edit event was successful